io/disk/lvsetup.py

```diff
@@ -45,6 +47,11 @@ def setUp(self):
         """
         Check existence of input PV,VG, LV and snapshots prior to Test.
         """
+        smm = SoftwareManager()
+        pkgs = ['xfsprogs', 'btrfs-progs']
+        for pkg in pkgs:
+            if not smm.check_installed(pkg) and not smm.install(pkg):
+                self.skip("%s package is need to test" % pkg)
```

Comment:
These packages are not required for Rhel. So can I do a distro.detect and install these for Ubuntu OS?